stable/spring-cloud-data-flow/requirements.yaml

```diff
@@ -2,10 +2,15 @@ dependencies:
 - name:  mysql
   version: 1.1.0
   repository: https://kubernetes-charts.storage.googleapis.com/
+  condition: mysql.enabled
 - name:  rabbitmq
   version: 0.6.16
   repository: https://kubernetes-charts.storage.googleapis.com/
   condition: rabbitmq.enabled
+- name:  rabbitmq-ha
+  version: 1.27.1
+  repository: https://kubernetes-charts.storage.googleapis.com/
+  condition: rabbitmq-ha.enabled
```

Comment:
are there any notes to indicate when setting `rabbitmq-ha.enabled` to `true`, `rabbitmq.enabled` should be set to `false`? otherwise id assume both would be deployed given `rabbitmq.enabled` defaults to `true` ?